.github/workflows/deploy_env.yml

```diff
@@ -39,3 +39,37 @@ jobs:
           cluster: ${{ secrets.KUBE_CLUSTER }}
           namespace: ${{ secrets.KUBE_NAMESPACE }}
           token: ${{ secrets.KUBE_TOKEN }}
+        continue-on-error: true
+
+      # Notification bit - always send prod releases to dps-releases - CVA3MKDTR
+      - if: ${{  inputs.environment == 'prod' || inputs.environment == 'production' }} 
+        id: prod-dps-slack
+        uses: ministryofjustice/hmpps-github-actions/.github/actions/slack_release_results@v1 # WORKFLOW_VERSION 
+        with:
+          channel_id: 'CVA3MKDTR'
+          environment: ${{ inputs.environment }}
+          app_version: ${{ inputs.app_version }}
+          deploy_outcome: ${{ steps.deploy.outcome }}
+          slack_bot_token: ${{ secrets.HMPPS_SRE_SLACK_BOT_TOKEN }}
+
+      
+      # Optional prod releases slack channel
+      - if: ${{ ( inputs.environment == 'prod' || inputs.environment == 'production' ) && vars.PROD_RELEASES_SLACK_CHANNEL != '' }}
+        id: prod-slack
+        uses: ministryofjustice/hmpps-github-actions/.github/actions/slack_release_results@v1 # WORKFLOW_VERSION 
+        with:
+          channel_id: ${{ vars.PROD_RELEASES_SLACK_CHANNEL }}
+          environment: ${{ inputs.environment }}
+          app_version: ${{ inputs.app_version }}
+          deploy_outcome: ${{ steps.deploy.outcome }}
+          slack_bot_token: ${{ secrets.HMPPS_SRE_SLACK_BOT_TOKEN }}
+
+      - if: ${{ (inputs.environment != 'prod' && inputs.environment != 'production') &&  vars.NONPROD_RELEASES_SLACK_CHANNEL != '' }}
```

Comment:
Maybe a comment here - ```# Optional non-prod releases slack channel```

Reply:
I've added this comment and also implemented a little icon
![image](https://github.com/user-attachments/assets/d5d50444-da13-4ac8-ad31-932788bf27e7)